manifests/init.pp

```diff
@@ -83,6 +83,9 @@
 # [*manage_archive*]
 #   Boolean.  Whether or not to manage InfluxData's tar archive.
 #
+# [*manage_user*]
+#   Boolean.  Whether or not to manage the 'telegraf' user when installing from archive.
```

Comment:
for new parameters we would like to have the puppet-strings style (that allows us to autogenerate documentation):
```suggestion
#   @param manage_user Whether or not to manage the 'telegraf' user when installing from archive.
```

Reply:
Yeah, with or without the new line, using `@param` is better :+1:.  The top of the comment also need to be adjusted to remove the markdown syntax (is it markdown? Y mean the `=` strings are no something puppet-string likes).

Puppet has some guidance for documenting classes here:
https://puppet.com/docs/puppet/7/puppet_strings_style.html#puppet_strings_style-strings-style-classes-types

Reply:
Done.  I've become reliant on `pdk validate` for this, I guess.